packages/cache/src/index.js

```diff
@@ -164,7 +164,9 @@ let createCache = (options: Options): EmotionCache => {
         }
       }
 
-      stylis(selector ? `${selector}{${serialized.styles}}` : serialized.styles)
+      stylis(
+        selector ? `${selector}{${serialized.styles};}` : serialized.styles
```

Comment:
this is the core of the fix - `}` is allowed in css variables so it was parsed as part of it and the whole thing resulted in incorrect styles as the opaque object does not contain a trailing `;`